cloudbuild/run-presubmit-on-k8s.sh

```diff
@@ -12,13 +12,22 @@ gcloud container clusters get-credentials "${CLOUDSDK_CONTAINER_CLUSTER}"
 
 LOGS_SINCE_TIME=$(date --iso-8601=seconds)
 
-kubectl run "${POD_NAME}" \
-  --image="${IMAGE}" \
-  --requests='cpu=750m,memory=2Gi,ephemeral-storage=2Gi' \
-  --restart=Never \
-  --env="COMMIT_SHA=${COMMIT_SHA}" \
-  --env="IMAGE_VERSION=${DATAPROC_IMAGE_VERSION}" \
-  --command -- bash /init-actions/cloudbuild/presubmit.sh
+#kubectl run "${POD_NAME}" \
```

Comment:
We should keep this old command in the previous commit in the main branch.  Thank you for preserving the previous implementation in comments.  I loathe the concept of removing the old implementation as well.  We could leave this comment in for the next release for posterity and then remove them to reduce file size and execution time.